### PR TITLE
Support api-data.line.me

### PR DESCRIPTION
**Pull request by kazuki-ma**
* Introduce `LIneBlobClient`.
* Property type migrated from `String` to `URI` (Basically no one configure this endpoint)

### CODE REVIEW

**Comment by kazuki-ma** [comment]
@sdk-java-dev This PR is ready for review. Please review it.

File: line-bot-spring-boot/src/main/java/com/linecorp/bot/spring/boot/LineBotProperties.java [hunk 1]
```diff
@@ -53,7 +53,11 @@
      */
     private @Valid @NotNull String channelSecret;
 
-    private @Valid @NotNull String apiEndPoint = LineClientConstants.DEFAULT_API_END_POINT;
+    /** blobEndPoint. default = https://api.line.me */
```
Comment (tkgauri) [inline line-bot-spring-boot/src/main/java/com/linecorp/bot/spring/boot/LineBotProperties.java]: [NITS] s/blobEndPoint/apiEndPoint/

File: line-bot-api-client/src/test/java/com/linecorp/bot/client/LineBlobClientImplTest.java [hunk 2]
```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright 2016 LINE Corporation
```
Comment (imasahiro) [inline line-bot-api-client/src/test/java/com/linecorp/bot/client/LineBlobClientImplTest.java]: 2016 -> 2019

**Comment by kazuki-ma** [comment]
Thank you for your reviewing. I updated PR. Please review again.

File: line-bot-api-client/src/test/java/com/linecorp/bot/client/AbstractWiremockTest.java [hunk 3]
```diff
@@ -66,14 +68,22 @@ public void mocking(final int responseCode, final ErrorResponse errorResponse) {
...
+                             .build();
+    }
+
     protected ChannelManagementSyncClient createChannelManagementSyncClient(final MockWebServer mockWebServer) {
         return ChannelManagementSyncClient
                 .builder(() -> "token")
                 .apiEndPoint(URI.create("http://localhost:" + mockWebServer.getPort()))
                 .build();
     }
+
```
Comment (imasahiro) [inline line-bot-api-client/src/test/java/com/linecorp/bot/client/AbstractWiremockTest.java]: nit. revert this line

Comment (kazuki-ma) [reply line-bot-api-client/src/test/java/com/linecorp/bot/client/AbstractWiremockTest.java]: 🙇 thank you.

**Pull request merged**